test/unit/modules/validation/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "modules":{
+    "validation":{
+      "enabled":true,
+      "implementation":{
+        "merkle-validation":{
+          "enabled":true,
+          "package":"./validation/implementation/merkle-validation.js",
+          "config":{
+            "database":"operationaldb-test",
```

Comment:
You can remove this part of configuration since it belongs to repository module not validation module